### PR TITLE
server: rename IContext to QueryCtx

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by coocood** [comment]
Update comment to fix CI

File: server/driver.go [hunk 1]
```diff
@@ -22,11 +22,11 @@ import (
 // IDriver opens IContext.
 type IDriver interface {
 	// OpenCtx opens an IContext with connection id, client capability, collation and dbname.
-	OpenCtx(connID uint64, capability uint32, collation uint8, dbname string) (IContext, error)
+	OpenCtx(connID uint64, capability uint32, collation uint8, dbname string) (QueryCtx, error)
 }
 
 // IContext is the interface to execute command.
-type IContext interface {
+type QueryCtx interface {
```
Comment (shenli) [inline server/driver.go]: Update the comment.

**Comment by ngaut** [comment]
PTAL

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**